### ISSUE TITLE
overConstrainedError on Chrome only

### ISSUE THREAD

**Issue by harinair-1** [twilio/twilio-video.js#1473]
**Code to reproduce the issue:**

```js
connect(this.token, {
              name: this.roomId,
              networkQuality: { local: 1, remote: 1 },
              video: {
                width: { exact: 1280},
                height: { exact: 720 },
              },
              maxAudioBitrate: 16000,
              bandwidthProfile: {
                video: {
                  mode: 'collaboration',
                },
              },
            });
```

**Expected behavior:**

HD video tracks are published and successfully subscribed to on all browsers.

**Actual behavior:**

When the min/exact dimensions are set to anything greater than 640x480, Chrome and Edge throw the following error: 

2021-05-12T09:33:15.832Z warn [createLocalTracks #5] Call to getUserMedia failed: OverconstrainedError {name: "OverconstrainedError", message: "", constraint: "width"}

The code works perfectly fine and all media is published when using Safari or Firefox, and works on Chrome and edge when the min/exact dimensions are set to 640x480

I have tried not using min/max/exact, and just specified the height and width, which results in Firefox and Safari displaying the track at HD resolution, and Chrome at VGA resolution.

**Software versions:**

 - Browser(s): Google Chrome v89.0.4389.90, Microsoft Edge v90.0.818.56
 - Operating System: Mac OS 11.2.2
 - twilio-video.js: 2.11.0
 - Third-party libraries: VueJS

**Comment by PikaJoyce** [comment]
Hey there @harinair-1 !

Thanks for submitting this issue.

I was able to reproduce this with your code sample on our Quickstart.

I was wondering if you could try passing in your desired height and width values like :
```
  video: { height: 720, width: 1280 }
```
And see if this works for you?

Here is the example we use in our [Quickstart](https://github.com/twilio/video-quickstart-js/blob/master/quickstart/src/index.js#L47). Passing in the same width and height.

Regards,
Joyce

**Comment by harinair-1** [comment]
Hi Joyce,

Thanks for getting back to me. I had previously tried it the way you mentioned and that still threw an overContrainedError. However after digging through some stuff I managed to figure out what was going on. 

It was in fact related to this bug in Chrome/Chromium [https://bugs.chromium.org/p/chromium/issues/detail?id=943469&can=1&q=From%20getCapabilities%20mikael&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified](getUserMedia constraints does not correspond to capabilities). I was previously activating a preview window with a lower resolution, and Chrome cannot change the resolution once the stream is active in the window.

All I had to do was make sure I used the max resolution possible every step of the way.

Thanks again!

Hari
